tests/src/main/java/com/hazelcast/stabilizer/tests/map/SSLEnterpriseTest.java

```diff
@@ -0,0 +1,104 @@
+package com.hazelcast.stabilizer.tests.map;
+
+
+import com.hazelcast.core.HazelcastInstance;
+import com.hazelcast.core.IMap;
+import com.hazelcast.logging.ILogger;
+import com.hazelcast.logging.Logger;
+import com.hazelcast.stabilizer.test.TestContext;
+import com.hazelcast.stabilizer.test.TestRunner;
+import com.hazelcast.stabilizer.test.annotations.Run;
+import com.hazelcast.stabilizer.test.annotations.Setup;
+import com.hazelcast.stabilizer.test.annotations.Teardown;
+import com.hazelcast.stabilizer.test.annotations.Verify;
+import com.hazelcast.stabilizer.test.utils.ThreadSpawner;
+import com.hazelcast.stabilizer.tests.helpers.KeyLocality;
+import com.hazelcast.stabilizer.tests.helpers.KeyUtils;
+import org.apache.commons.lang3.ArrayUtils;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Random;
+
+import static com.hazelcast.stabilizer.test.utils.TestUtils.assertEqualsByteArray;
+
+public class SSLEnterpriseTest {
+
+    private final static ILogger log = Logger.getLogger(SSLEnterpriseTest.class);
+    private static Random random = new Random();
+
+    //properties
+    public String basename = "sslEnterprise";
+    public int threadCount = 3;
+    public int keyCount = 1000;
+    public int keyLength = 5;
+    public KeyLocality keyLocality = KeyLocality.Local;
```

Comment:
Could be me, but if all keys are local.. are you still testing remote communication? (And therefore SSL)